File: Upcoder/Khối 11/Chương 1 Làm Quen/Tam giác/K11_TamGiac.cs
using System;
namespace Tamgiac
{
    public class Program
    {
        static void Main(string[]args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int n = input.Length;
            double []Arr = new double[n];
            for(int i = 0; i < n; i++)
                Arr[i] = double.Parse(input[i]);
            double a = Arr[0];
            double b = Arr[1];
            double C = Arr[2];
            if(C == 180)
            {
                Console.Write("KHONG PHAI TAM GIAC");
            }
            else
            {
                C = C * Math.PI/ 180.0;
                double c = Math.Sqrt(Math.Pow(a,2) + Math.Pow(b,2) -2 * a * b * Math.Cos(C));
                Console.Write(c);
            }
        }
    }
}

File: Upcoder/Khối 11/Chương 2 Cấu trúc rẽ nhánh/Tam giác/K11_TamGiac.py
import math

a ,b ,C = map(float, input().split())
if(C == 180):
    print("KHONG PHAI TAM GIAC", end ="")
else:
    C *= math.pi / 180
    c = math.sqrt( a * a + b * b - 2 * a * b * math.cos(C))
    print(c)